### DISCUSSION TITLE
Include more details at lower zoom level

### DISCUSSION THREAD

**Discussion by rlightner** [onthegomap/planetiler#1002] [General]
I would like to include the streams at zoom level 12, that are shown at zoom level 13. How can I make this happen?
![image](https://github.com/user-attachments/assets/e79fa95e-9f0a-45c2-8ddc-def5d80b42af) ![image](https://github.com/user-attachments/assets/d1d7d8ba-e631-41a8-9b1d-529944d2266e)
Thanks!

**Comment by wipfli** [comment]
Planetiler can be used for different vector tile schemas like OpenMapTiles, Protomaps, etc. if your question is about OMT you might look closer here:

https://github.com/openmaptiles/planetiler-openmaptiles/blob/74a4fbec2564d13f6713ef8baaf277eb86bf456f/src/main/java/org/openmaptiles/generated/OpenMapTilesSchema.java#L268
